src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialService.scala

```diff
@@ -222,7 +222,7 @@ final class TrialService
 
   private def verifyProjects: Future[PerRequestMessage] = {
 
-    val saToken:WithAccessToken = AccessToken(OAuth2BearerToken(HttpGoogleServicesDAO.getTrialBillingManagerAccessToken))
+    val saToken:WithAccessToken = AccessToken(OAuth2BearerToken(googleDAO.getTrialBillingManagerAccessToken))
```

Comment:
it was necessary to use the google dao from the constructor instead of hardcoding the HttpGoogleServicesDAO in order for the unit tests to allow mocks.